lib/resources/example_json/r4_examples_insurance_plan.rb

```diff
@@ -0,0 +1,141 @@
+# frozen_string_literal: true
+
+module Cerner
+  module Resources
+    R4_INSURANCEPLAN_BUNDLE ||= {
+      'resourceType': 'Bundle',
+      'id': 'e8b5a95f-c125-4509-ac99-34e6f9b6639f',
+      'type': 'searchset',
+      'total': 3,
+      'link': [
+        {
+          'relation': 'self',
+          'url': 'https://fhir.applications.stagingcerner.com/r4/9ce2bac1-f00e-43d9-ab88-89e1d4c52e1a/InsurancePlan?owned-by=589743'
```

Comment:
It looks like we are using the staging url in all the places. I believe it should be a production url instead. Please double check.

Reply:
It Should be a prod URL but since we do not have it in prod yet what would you suggest @BhavyaAcharya

Reply:
We can have Sandbox data for now, and update the examples once released to prod

Reply:
We want the url to still be from prod/c1941 https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/InsurancePlan?owned-by=589743 . We can update the organization id and example response later.

Reply:
Can you please update it to open endpoint, like https://fhir-open.cerner.com/ ..

Reply:
Done https://github.com/cerner/fhir.cerner.com/pull/809/commits/b15a0dd161ba774ee4a02ba6c8fc2d398615a7f3#diff-505b119a76d7cbd4b9ad19f605bcfd1d674bfb0fca50632ebc7948b1bc50b3caR52